src/main/kotlin/org/malachite/estella/organization/api/OrganizationController.kt

```diff
@@ -23,21 +26,22 @@ class OrganizationController(@Autowired private val organizationService: Organiz
     @GetMapping("/{organizationId}")
     fun getOrganization(@PathVariable organizationId: OrganizationID): ResponseEntity<Organization> {
         val organization: Organization = organizationService.getOrganization(organizationId.toId())
-
-        return  ResponseEntity(organization, HttpStatus.OK)
+        return ResponseEntity(organization, HttpStatus.OK)
```

Comment:
```kotlin
fun ...(...) = 
    organizationService.getOrganization(organizationId.toId())
        .let { ResponseEntity(organization, HttpStatus.OK) }
```
pamiętajmy, że kotlin ma ogólnie te fajne funkcje, po których możesz dać = i robi Ci to returna. Nie trzeba wtedy dawać zwracanego typu, co jest spoko + pozbędziemy się tej zmiennej organization.